subsys/bluetooth/controller/ll_sw/isoal.c

```diff
@@ -1457,10 +1457,17 @@ static isoal_status_t isoal_tx_pdu_emit(const struct isoal_source *source_ctx,
 	return status;
 }
 
-/* Allocates a new PDU only if the previous PDU was emitted */
+/**
+ * Allocates a new PDU only if the previous PDU was emitted
+ * @param[in]  source      ISO-AL source reference
+ * @param[in]  tx_sdu      SDU fragment to be transmitted (can be NULL)
```

Comment:
Since `tx_sdu` is unused, shouldn't that be mentioned here?

Reply:
Personally I would prefer not to do that. The reason is I view the header as describing the interface to the function not it's implementation and if the implementation changes without changing the interface, then updating the header is likely to be overlooked. I think the ARG_UNUSED at the first line is sufficient indication of what the implementation is at present.

Reply:
I have no strong opinions here. Since this is documentation for a `static` function, either way works